webrtc/videoroom/.env.example

```diff
@@ -0,0 +1,7 @@
+VIRTUAL_HOST=localhost
```

Comment:
Do we want it to be named `.env.example`? Why not `.env`? It's demo so I would put it as `.env` and remove instruction to copy `.env.example` to `.env` in README.